css/project-page.styl

```diff
@@ -1,3 +1,21 @@
+.about-container
+  background: rgba(255,255,255,0.95)
+  border-radius: 3px
+  color: #000000
+  margin: 1.5em 12vw
+  padding: 1.5em 6vw
+
+  img
+    max-width: 100%
+
+  h1:first-child, h2:first-child,
```

Comment:
This was a bit tricky. Basically, according to the template I received from Jordan, only the first header would be colored. Since this content is rendered in Markdown, this seemed to be the best way to only style the first header in that section.